src/java/org/apache/cassandra/index/sai/disk/v1/segment/SegmentBuilder.java

```diff
@@ -161,18 +154,13 @@ private void copyBufferToBytesRef(ByteBuffer buffer, BytesRefBuilder stringBuffe
         }
     }
 
-    public static int getActiveBuilderCount()
-    {
-        return ACTIVE_BUILDER_COUNT.get();
-    }
-
-    private SegmentBuilder(AbstractType<?> termComparator, NamedMemoryLimiter limiter)
+    private SegmentBuilder(AbstractType<?> termComparator)
     {
         this.termComparator = termComparator;
-        this.limiter = limiter;
+//        this.limiter = limiter;
```

Comment:
TODO: cleanup, depending on how we finalize the structure of the limiter class/singleton